src/storage.ts

```
@@ -0,0 +1,90 @@
+      const config = this.getConfig();
+      const lastUpdated = new Date(config.lastUpdated);
+      const now = new Date();
+      return now.getTime() - lastUpdated.getTime();
+    }
+  
+    static isConfigRecent(maxAgeHours: number = 24): boolean {
+      const ageMs = this.getConfigAge();
+      const maxAgeMs = maxAgeHours * 60 * 60 * 1000;
+      return ageMs < maxAgeMs;
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)